packages/lesswrong/components/common/ContentItemBody.tsx

```diff
@@ -1,92 +1,70 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
 import { Components, registerComponent } from '../../lib/vulcan-lib';
-import classNames from 'classnames';
 import { captureException }from '@sentry/core';
-import { isServer } from '../../lib/executionEnvironment';
 import { linkIsExcludedFromPreview } from '../linkPreview/HoverPreviewLink';
 import { isEAForum } from '../../lib/instanceSettings';
-import withUser from './withUser';
-import { withLocation } from '../../lib/routeUtil';
-
-const styles = (theme: ThemeType): JssStyles => ({
-  scrollIndicatorWrapper: {
-    display: "block",
-    position: "relative",
-    
-    paddingLeft: 13,
-    paddingRight: 13,
-  },
-  
-  hidden: {
-    display: "none !important",
-  },
-  
-  scrollIndicator: {
-    position: "absolute",
-    top: "50%",
-    marginTop: -28,
-    cursor: "pointer",
-    
-    // Scroll arrows use the CSS Triangle hack - see
-    // https://css-tricks.com/snippets/css/css-triangle/ for a full explanation
-    borderTop: "20px solid transparent",
-    borderBottom: "20px solid transparent",
-  },
-  
-  scrollIndicatorLeft: {
-    left: 0,
-    borderRight: `10px solid ${theme.palette.grey[310]}`,
-    
-    "&:hover": {
-      borderRight: `10px solid ${theme.palette.grey[620]}`,
-    },
-  },
-  
-  scrollIndicatorRight: {
-    right: 0,
-    borderLeft: `10px solid ${theme.palette.grey[310]}`,
-    
-    "&:hover": {
-      borderLeft: `10px solid ${theme.palette.grey[620]}`,
-    },
-  },
-  
-  scrollableLaTeX: {
-    // Cancel out the margin created by the block elements above and below,
-    // so that we can convert them into padding and get a larger touch
-    // target.
-    // !important to take precedence over .mjx-chtml
-    marginTop: "-1em !important",
-    marginBottom: "-1em !important",
-    
-    paddingTop: "2em !important",
-    paddingBottom: "2em !important",
-    
-    // Hide the scrollbar (on browsers that support it) because our scroll
-    // indicator is better
-    "-ms-overflow-style": "-ms-autohiding-scrollbar",
-    "&::-webkit-scrollbar": {
-      display: "none",
-    },
-    scrollbarWidth: "none",
-  },
-
-});
+import domAnchorTextQuote from 'dom-anchor-text-quote';
+import { rawExtractElementChildrenToReactComponent, wrapRangeWithSpan } from '../../lib/utils/rawDom';
 
 interface ExternalProps {
+  /**
+   * The content to show. This MUST come from a GraphQL resolver which does 
+   * sanitization, such as post.contents.html
+   */
   dangerouslySetInnerHTML: { __html: string };
+  
+  /**
+   * Type-annotation reflecting that you can make a ref of this and call its
```

Comment:
fwiw I had trouble parsing this comment.

Reply:
is "reflecting" a technical term here or just colloquially means... something? (either way I wasn't quite sure what it meant)

Reply:
just "indicating"